### DISCUSSION TITLE
Something's wrong

### DISCUSSION THREAD

**Discussion by Reepzvan25** [Shabinder/SpotiFlyer#101] [Show and tell]
Hello to the community newbie here. 
The previous version is good it can download songs and albums and etc. The latest version, yes it can download the songs but it won't play in my device, when I delete all the tracks I noticed the file size of the track is small. Like 10 songs all of that is size 4.30 mb.

**Comment by Shabinder** [comment]
Yes , I have been notified about the issue in #97 , for now use v2.1 , I will be releasing a new update with the fix soon

**Comment by Reepzvan25** [comment]
Also the 2.2.0 version that I'm using right now I already been using this
last months it's working fine bute when i update it zoinks hehe.

**Reply by Shabinder** [reply]
> Also the 2.2.0 version that I'm using right now I already been using this last months it's working fine bute when i update it zoinks hehe.
> […](#)
> On Thu, Apr 29, 2021, 11:05 PM Shabinder Singh, ***@***.***> wrote: Yes , I have been notified about the issue in #97 <#97> , for now use v2.1 , I will be releasing a new update with the fix soon — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub <[#101 (comment)](https://github.com/Shabinder/SpotiFlyer/discussions/101#discussioncomment-675348)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AT4MWARKKUWJWYGS44NYFBLTLFYUNANCNFSM432B3ZRQ> .

I know 2.2.0 was working fine , but it stopped 2-3 days earlier , So use 2.1 till I release new update
